libs/logging/src/main/java/org/elasticsearch/logging/internal/spi/LogManagerFactory.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.logging.internal.spi;
+
+import org.elasticsearch.logging.Logger;
+
+/**
+ * SPI for creating new loggers
```

Comment:
not really SPI anymore, it's just a static holder for the factory

Reply:
but doesn't it still serve a purpose of SPI? the implementation has to provide the concrete getLogger methods that this class declares

Reply:
SPI normally has the connotation of being loaded by ServiceProvider, but this is not.